django_token_bucket/models.py

```diff
@@ -34,14 +34,16 @@ def __str__(self, tz=None):
 
 class TokenBucket(models.Model):
     identifier = models.CharField(max_length=30)
-    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, null=True)
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    object_id = models.PositiveIntegerField()
+    user = GenericForeignKey('content_type', 'object_id')
```

Comment:
This field should not be called `user`.

Additionally, according to the docs, the parameters can be omitted if we use the default field names (which are `content_type` and `object_id`)